<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddQuestion</class>
 <widget class="QWidget" name="AddQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑题目</string>
  </property>
  <layout class="QGridLayout" name="layoutGWidget">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxQuestion">
     <property name="title">
      <string>问题</string>
     </property>
     <layout class="QGridLayout" name="layoutGroupQuestion">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="4" column="1">
       <widget class="QLabel" name="labelC">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelD">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="labelOptions">
        <property name="text">
         <string>选项</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelDesc">
        <property name="text">
         <string>描述</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelB">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QLabel" name="labelA">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="5">
       <widget class="QPlainTextEdit" name="textDesc"/>
      </item>
      <item row="3" column="2">
       <widget class="QPlainTextEdit" name="textB"/>
      </item>
      <item row="4" column="2">
       <widget class="QPlainTextEdit" name="textC"/>
      </item>
      <item row="5" column="2">
       <widget class="QPlainTextEdit" name="textD"/>
      </item>
      <item row="2" column="2">
       <widget class="QPlainTextEdit" name="textA"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="layoutHInfo">
     <item>
      <widget class="QLabel" name="labelSubject">
       <property name="text">
        <string>科目：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSubject"/>
     </item>
     <item>
      <widget class="QLabel" name="labelChapter">
       <property name="text">
        <string>章节：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxChapter">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerHInfo">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="layoutHBtns">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <spacer name="spacerHBtns">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancer">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxAnswer">
     <property name="title">
      <string>参考答案</string>
     </property>
     <layout class="QGridLayout" name="layoutGAnswer">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="0" column="2">
       <spacer name="spacerHAnswer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelAnswer">
        <property name="text">
         <string>答案：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxAnswer">
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="labelDetail">
        <property name="text">
         <string>详解</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QPlainTextEdit" name="textDetail"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
